contribs/gmf/src/layertree/component.js

```diff
@@ -525,6 +525,20 @@ Controller.prototype.getNodeState = function (treeCtrl) {
   return treeCtrl.getState();
 };
 
+/**
+ * Update the `timeRangeValue` property of the data source bound to the
+ * given tree controller and the state of the permalink using the given time.
+ *
+ * LayertreeController.prototype.updateTimeData - description
+ * @param {import("ngeo/layertree/Controller.js").LayertreeController} layertreeCtrl ngeo layertree controller
+ * @param {{start : number, end : number}} time The start
```

Comment:
```suggestion
 * @param {import("ngeo/datasource/OGC.js").TimeRange} time The start
```
It should works. Please replace also the other definitions.